loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowResult.java

```diff
@@ -47,22 +85,44 @@ enum Status {
          */
         Status getStatus();
 
+        /**
+         * Get metrics. Metrics are generic key/value pairs and are specific to a loadflow implementation.
+         * @return the metrics for this component
+         */
+        Map<String, String> getMetrics();
+
         /**
          * Get iteration count.
          * @return the iteration count
          */
         int getIterationCount();
 
+        /**
+         * Get the reference bus id (angle reference)
+         * @return the reference bus id
+         */
+        String getReferenceBusId();
+
+        /**
+         * get slack results of all slack buses
+         * @return list of slack results
+         */
+        List<SlackResult> getSlackResults();
```

Comment:
Just a detail but we provide a list because we can have several slack buses. So why you don't call it `SlackBusResult` and inside a `SlackBusResut`, you just provide a `getId()` and a `getActivePowerMismatch()` ?

Reply:
good catch, will change this